src/main/java/com/github/krzysiekjodlowski/ox/model/Symbol.java

```diff
@@ -0,0 +1,26 @@
+package com.github.krzysiekjodlowski.ox.model;
+
+/**
+ * Represents each player on Board.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public enum Symbol {
+  NAUGHT("O"), CROSS("X"), EMPTY(" ");
```

Comment:
Do you need EMPTY when you store marked fields in map?

Reply:
I used it as a placeholder in a few places but you're right - removed.